examples/vue2/package.json

```diff
@@ -14,7 +14,7 @@
     "less": "4.1.3",
     "less-loader": "^11.1.0",
     "vue": "2.7.14",
-    "vue-loader": "^15.10.1",
+    "vue-loader": "npm:@rspack/loader-vue2@15.10.1-alpha.0",
```

Comment:
Can @rspack/loader-vue2 release a stable version?

Reply:
I'm not sure, cc @h-a-n-a , and the author file PR to vue-loader repo, when it's merged we can replace it with official vue-loader

Reply:
I don't know if it is a good choice to file a PR to vue, but seems like the target branch should be set to v15

Reply:
I  talked with soda about this, he's willing to accept the PR

Reply:
Reviewed and published as https://github.com/vuejs/vue-loader/releases/tag/v15.11.0

Reply:
Thanks!